functional_tests/steps/statistical_article_page.py

```diff
@@ -174,6 +175,55 @@ def the_published_statistical_article_page_has_the_added_correction(context: Con
     expect(context.page.get_by_text("Correction text")).to_be_hidden()
 
 
+@then("the user can expand and collapse {block_type} details")
+def user_can_click_on_view_detail_to_expand_block(context: Context, block_type: str):
+    if block_type == "correction":
+        text = "Correction text"
+        date = "13 March 2025 1:59p.m."
+    else:
+        text = "Notice text"
+        date = "15 March 2025 1:59p.m."
+
+    context.page.get_by_text("Show detail").click()
+    expect(context.page.get_by_text(text)).to_be_visible()
+    expect(context.page.get_by_text(date)).to_be_visible()
+    if block_type == "notice":
```

Comment:
Shouldn't this be "correction"? As a superseded version only exists if there is a correction.

Reply:
🤦